dnf-behave-tests/features/repoinfo.feature

```diff
@@ -0,0 +1,116 @@
+Feature: Repoinfo
+
+
+Background: Using repositories dnf-ci-fedora and dnf-ci-thirdparty-updates
+  Given I use repository "dnf-ci-fedora"
+    And I use repository "dnf-ci-thirdparty-updates"
+    And I use repository "dnf-ci-fedora-updates" with configuration
+        | key     | value |
+        | enabled | 0     |
+    And I use repository "dnf-ci-thirdparty" with configuration
+        | key     | value |
+        | enabled | 0     |
+
+
+Scenario: Repolist without arguments
+   When I execute dnf with args "repoinfo"
+   Then the exit code is 0
+    And stdout is
+"""
+Repo-id            : dnf-ci-fedora
+Repo-name          : dnf-ci-fedora test repository
+Repo-revision      : 1550000000
+Repo-updated       : Mon Nov 18 20:00:44 2019
+Repo-pkgs          : 289
+Repo-available-pkgs: 289
+Repo-size          : 2.0 M
+Repo-baseurl       : file:///opt/behave/fixtures/repos/dnf-ci-fedora
```

Comment:
although this url could be correct when run in the docker container, if you run the test locally you would get different path - in my case `file:///home/mblaha/src/ci-dnf-stack/dnf-behave-tests/fixtures/repos/dnf-ci-fedora`